omero/sysadmins/config.rst

```diff
@@ -2210,7 +2210,7 @@ Default: `None`
 
 omero.web.login_redirect
 ^^^^^^^^^^^^^^^^^^^^^^^^
-Redirect to the given location after logging in. It only supports arguments for :djangodoc:`Django reverse function <ref/urlresolvers/#django.core.urlresolvers.reverse>`. For example: ``'{"redirect": ["webindex"], "viewname": "load_template", "args":["userdata"], "query_string": {"experimenter": -1}}'``
+Redirect to the given location after logging in. It only supports arguments for :djangodoc:`Django reverse function <ref/urlresolvers/#reverse>`. For example: ``'{"redirect": ["webindex"], "viewname": "load_template", "args":["userdata"], "query_string": {"experimenter": -1}}'``
```

Comment:
@will-moore : these will need back porting to omero-web.

Reply:
Done - see https://github.com/ome/omero-web/pull/103/files#diff-aa8789d92bc0441643a6535447739dbd

Reply:
Ah, ta. Then I should have done an OMERO.web release for the 5.6.0-rc1 docs. Sorry about that.